app/.meteor/.finished-upgraders

```diff
@@ -15,3 +15,4 @@ notices-for-facebook-graph-api-2
 1.4.1-add-shell-server-package
 1.4.3-split-account-service-packages
 1.5-add-dynamic-import-package
+1.7-split-underscore-from-meteor-base
```

Comment:
This should be part of the 1.8 update. Lets defer until after 1.8, if that is alright.